reference/v8.json

```diff
@@ -1241,43 +1359,64 @@
   },
   "filter_operator": {
     "type": "enum",
-    "values": [
-      "==",
-      "!=",
-      ">",
-      ">=",
-      "<",
-      "<=",
-      "in",
-      "!in",
-      "all",
-      "any",
-      "none",
-      "has",
-      "!has"
-    ],
+    "values": {
+      "==": {
+          "doc": "The equal-to operator."
```

Comment:
I think this is pretty clear in the context of GL styling, or if not, is very easily fixed (flip things, using the format `key == value`).